docs/topics_5/web-create-project.adoc

```diff
@@ -0,0 +1,111 @@
+
+ . Click *Next*.
++
+.Transformation target screen
+image::web-configure-analysis_51.png[Transformation target screen]
+
+. Click the desired transformation targets.
+. Click *Next*.
++
+You can specify which Java application packages to include or exclude on the *Select applications* screen.
++
+The list on the left contains all of the identified packages in the project. The list on the right contains only the packages that are to be analyzed.
+
+. Select the packages you want to include or exclude and click *>* or *<*.
+. Click *Next*.
++
+.Custom rules screen
+image::web-custom-rules-project_51.png[Custom rules screen]
++
+You can add custom rules to use in each project by uploading the custom rules or registering a server path that contains the rules.
+
+. If you want to add a rule, click *Add rule*.
+. To upload rules, click *Browse*, select a rule set file (`\*.mta.xml` or `*.windup.xml`), and click *Close*.
++
+Uploading rules stores them directly in the {ProductShortName} server.
++
+. To register a server path, click the *Server path* tab, enter the *Rules path* of the rules file in the field, and click *Save*.
++
+Registering the server path of a file tells the {ProductShortName} server the location of the rules. This method is useful if you expect to update the rules because it ensures that the {ProductShortName} server always uses the latest version of the rules.
++
+[NOTE]
+====
+For information on writing custom {ProductShortName} rules, see the {ProductShortName} link:{ProductDocRulesGuideURL}[_{RulesDevBookName}_].
+====
++
+. Click *Next*.
++
+.Custom labels screen
+image::web-custom-labels-project_51.png[Custom labels screen]
++
+You can add custom labels to use in each project by uploading labels or registering a server path that contains a label set.
++
+. If you want to add a label, *Add label*.
+. To upload labels, click *Browse*, select a label file (`\*.mta.label.xml` or `*.windup.label.xml`), and click *Close*.
++
+Uploading labels stores them directly in the {ProductShortName} server.
+
+. To register a server path, click the *Server path* tab, enter the *Labels path* of the label files in the field, and click *Save*.
++
+Registering the server path of a file tells the {ProductShortName} server the location of the labels. This method is useful if you expect to update the labels because it ensures that the {ProductShortName} server always uses the latest version of the labels.
++
+. Click *Next*.
++
+You can customize your analysis by setting some or all of the options on the *Advanced options* screen.
++
+. On the left side of the *Advanced options* screen, complete the following fields:
+
+* *Target*: Select from the list. By default, these are the targets you chose previously.
```

Comment:
Remove the "By default" sentence. The user can make the connection between selected targets and targets selected previously.